PyTDF/backend/Local.py

```diff
@@ -0,0 +1,31 @@
+import ROOT
+
+def local_executor(generator):
+    """
+    Execution of the event-loop
+    in local environment
+
+    """
+    mapper = generator.get_mapper_callable()
+
+    TChain = ROOT.TChain(generator.root_node.treename)
+    # TODO(shravan97) : Change the ctor to RDataFrame's
+
+    for f in generator.root_node.filelist:
+        TChain.Add(f)
+
+    TDF = ROOT.Experimental.TDataFrame(TChain)
+
+    output = mapper(TDF)
+
+    node_map = generator.action_node_map
+
+    event_loop_done = False
+
+    for n in node_map:
+        if not event_loop_done:
+            # Trigger event-loop
+            output[n].GetValue()
+            event_loop_done = True
```

Comment:
this only does something for the first ouput right? which is correct, but this is a weird way to write it :)

Reply:
So, shall I assign a new variable like `var = output[n]` and then do a `var.GetValue()` then ?

Reply:
first of all you should be sure that there is always at least one result, but I think that's the case.
given that, you can just do `output[0].GetValue()` to trigger the event loop.

Reply:
But @bluehood , the output of the callable is a dictionary and not a list. If it was a list, that would be good enough. But even in the dictionary, I know that the first key would be `ta1`. So I guess something like `output['ta1'].GetValue()` would be good enough ?

Reply:
ok ok, `output.values()[0].GetValue()`

Reply:
```python
output.values()[0].GetValue() # trigger event loop
```

Reply:
Sounds great. I'll fix this 👍

Reply:
@bluehood , fixed 👍